Index: python-daemon-2.0.6/test/test_daemon.py
===================================================================
--- python-daemon-2.0.6.orig/test/test_daemon.py
+++ python-daemon-2.0.6/test/test_daemon.py
@@ -32,7 +32,12 @@ except ImportError:
     # Standard library of Python 2.6 and earlier.
     from StringIO import StringIO
 
-import mock
+try:
+    # Python 3
+    from unittest import mock
+except ImportError:
+    # Python 2
+    import mock
 
 from . import scaffold
 from .scaffold import (basestring, unicode)
Index: python-daemon-2.0.6/test/test_metadata.py
===================================================================
--- python-daemon-2.0.6.orig/test/test_metadata.py
+++ python-daemon-2.0.6/test/test_metadata.py
@@ -29,7 +29,13 @@ import collections
 import json
 
 import pkg_resources
-import mock
+try:
+    # Python 3
+    from unittest import mock
+except ImportError:
+    # Python 2
+    import mock
+
 import testtools.helpers
 import testtools.matchers
 import testscenarios
Index: python-daemon-2.0.6/test/test_pidfile.py
===================================================================
--- python-daemon-2.0.6.orig/test/test_pidfile.py
+++ python-daemon-2.0.6/test/test_pidfile.py
@@ -33,7 +33,13 @@ except ImportError:
     # Standard library of Python 2.6 and earlier.
     from StringIO import StringIO
 
-import mock
+try:
+    # Python 3
+    from unittest import mock
+except ImportError:
+    # Python 2
+    import mock
+
 import lockfile
 
 from . import scaffold
Index: python-daemon-2.0.6/test/test_runner.py
===================================================================
--- python-daemon-2.0.6.orig/test/test_runner.py
+++ python-daemon-2.0.6/test/test_runner.py
@@ -30,7 +30,13 @@ import signal
 import functools
 
 import lockfile
-import mock
+try:
+    # Python 3
+    from unittest import mock
+except ImportError:
+    # Python 2
+    import mock
+
 import testtools
 
 from . import scaffold
